migration-params-ethglobal.json

```diff
@@ -0,0 +1,73 @@
+{
+    "orgName":"ETHGlobal",
+    "tokenName":"ETHGlobal",
+    "tokenSymbol":"ETG",
+    "ContributionReward": [
+        {
+            "voteParams":0
+        }
+    ],
+    "SchemeRegistrar": [
+        {
+            "voteRegisterParams": 1,
+            "voteRemoveParams": 1
+        }
+    ],
+    "CustomSchemes":[
+        {
+            "name": "ReputationFromToken",
+            "isUniversal": false,
+            "params": [
+              { "StandAloneContract": 1 },
```

Comment:
need to add https://github.com/daostack/arc/blob/master/contracts/utils/RepAllocation.sol 
as StandAloneContract